static/js/components/email/EmailCompositionDialog.js

```diff
@@ -67,6 +68,7 @@ type EmailDialogProps = {
   updateEmailFieldEdit:       () => void,
   renderRecipients?:          (filters: ?Array<Filter>) => React$Element<*>,
   updateEmailBody:            (e: Object) => void,
+  type:                       string
```

Comment:
can you rename this prop from `type` to `dialogType`? that's more specific, and `type` on its own can be confusing